paper-api/src/main/java/org/bukkit/event/player/PlayerHarvestBlockEvent.java

```diff
@@ -5,13 +5,16 @@
 import org.bukkit.entity.Player;
 import org.bukkit.event.Cancellable;
 import org.bukkit.event.HandlerList;
+import org.bukkit.event.entity.EntityHarvestBlockEvent;
 import org.bukkit.inventory.EquipmentSlot;
 import org.bukkit.inventory.ItemStack;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * This event is called whenever a player harvests a block.
  * <br>
+ * For the entity case please use {@link org.bukkit.event.entity.EntityHarvestBlockEvent}
```

Comment:
I think this line here and in the new event's javadoc could be a bit better, since there's an import it's also not necessary to use the fully qualified name.
```suggestion
 * For cases involving entities, please use {@link EntityHarvestBlockEvent}.
```